_posts/2018-12-19-map-get-and-update.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title:  "Elixir: Map 'get and update'"
+date:   2018-12-17 07:37:29 +0000
+categories: elixir
+author: Valerie Dryden
+---
+
+The other day I found myself trying to get the count of how many times an item appeared in a list in Elixir. There's no inbuilt way to do this that I've seen so far (please do comment if you know of one!) and there's various ways to achieve it but I chose to do it like so:
+
+```elixir
+defmodule Day4.Scratch do
```

Comment:
Question: is `Day4` a hint to what you're working on? Should we choose another name?

Reply:
Whoops! I was being lazy running it inside an existing mix project that had that namespace, good catch. Have removed.